src/Framework/UnitTests/Microsoft.Build.Framework.UnitTests.nuget.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
+    <NuGetPackageRoot>$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+  </PropertyGroup>
+  <ImportGroup>
+    <Import Project="$(NuGetPackageRoot)\xunit.core\2.1.0\build\portable-net45+win8+wp8+wpa81\xunit.core.props" Condition="Exists('$(NuGetPackageRoot)\xunit.core\2.1.0\build\portable-net45+win8+wp8+wpa81\xunit.core.props')" />
+    <Import Project="$(NuGetPackageRoot)\xunit.runner.visualstudio\2.1.0\build\net20\xunit.runner.visualstudio.props" Condition="Exists('$(NuGetPackageRoot)\xunit.runner.visualstudio\2.1.0\build\net20\xunit.runner.visualstudio.props')" />
+  </ImportGroup>
+</Project>
```

Comment:
This is a generated file that should be in .gitignore and not checked in, right?